### PR TITLE
Replace local script by `OpenTermsArchive/changelog-action`

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: .github/workflows/changelog.yml [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
   pull_request:
-    types: [ opened, reopened ]
+  workflow_dispatch:
   workflow_call:
     outputs:
-      release_type:
+      release-type:
         description: The release type extracted from changelog
-        value: ${{ jobs.check_changelog.outputs.release_type }}
+        value: ${{ jobs.validate_changelog.outputs.release-type }}
```
Comment (Ndpnt) [inline .github/workflows/changelog.yml:10]:
```suggestion
        value: ${{ jobs.validate.outputs.release-type }}
```

File: .github/workflows/changelog.yml [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
 
 jobs:
-  check_changelog:
-    uses: OpenTermsArchive/engine/.github/workflows/changelog.yml@v0.37.1
+  validate_changelog:
```
Comment (Ndpnt) [inline .github/workflows/changelog.yml:13]:
```suggestion
  validate:
```

**Review by Ndpnt — APPROVED** [review]

**Comment by clementbiron** [comment]
In consultation with the core team, I'm taking responsibility for bypassing the check_changelog test.

**Comment by MattiSG** [comment]
This test is required because the branch protection is not up to date in the repo settings. Let's just change that 🙂

**Pull request merged**